rules-java-archives/addon/src/main/java/org/jboss/windup/rules/apps/java/archives/identify/IdentifiedArchives.java

```diff
@@ -53,8 +53,7 @@ public static void addMappingsFrom(File file)
                     continue;
                 String[] parts = StringUtils.split(line, ' ');
                 if (parts.length < 2)
-                    throw new IllegalArgumentException("Expected 'SHA1 GROUP_ID:ARTIFACT_ID:VERSION[:COORDINATE]', but was [" + line + "] in ["
-                                + file + "]");
+                    throw new IllegalArgumentException("Expected 'SHA1 GROUP_ID:ARTIFACT_ID:[PACKAGING:[COORDINATE:]]VERSION', but was: \n" + line + "\n\tin [" + file + "]");
```

Comment:
Yeah, I was having a little fun with "grepping" since you talk about having a hard time finding things with grep, sorry :) Point is, there's really not much reason for this exception to span multiple lines. You're not comparing or enumerating anything. It just doesn't seem appropriate in this case.